mozilla-release/mobile/android/base/java/org/mozilla/gecko/db/GhosteryHistoryItem.java

```diff
@@ -0,0 +1,27 @@
+package org.mozilla.gecko.db;
+
+/**
+ * @author Ravjit Uppal
+ */
```

Comment:
We do not use `@author` in mozilla, when we create new files we just add the following copyright notice:

```java
/*
 * Copyright © Cliqz 2018
 */
```